content/en/blog/_posts/2022-11-28-advancements-in-traffic-engineering/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+title: "Kubernetes v1.26: Advancements in Kubernetes Traffic Engineering"
+date: 2022-11-28
+slug: advancements-in-kubernetes-traffic-engineering
+---
+
+**Authors:** Andrew Sy Kim (Google)
+
+Kubernetes v1.26 includes significant advancements in traffic engineering with the graduation of the
+`ServiceInternalTrafficPolicy` and `EndpointSliceTerminatingCondition` features to GA and the
+`ProxyTerminatingEndpoints` feature to Beta. The combination of these enhancements aim to address
+short-comings in traffic engineering today and unlock new capabilities for the future.
+
+## Traffic Loss from Load Balancers During Rolling Updates
+
+Prior to Kubernetes v1.26, [clusters can experience loss of traffic](https://github.com/kubernetes/kubernetes/issues/85643)
+from Service load balancers during rolling updates when setting the `externalTrafficPolicy` field to `Local`.
+There are a lot of moving parts at play here so a quick overview of how Kubernetes manages load balancers might help!
+
+In Kubernetes, you can create a Service with Type=LoadBalancer to expose an application externally with a load balancer.
```

Comment:
```suggestion
In Kubernetes, you can create a Service with `type: LoadBalancer` to expose an application externally with a load balancer.
```
YAML is the norm; JSON is a fallback. `Type=LoadBalancer` isn't part of the API either way (it's more like Golang).